tests/integ/test_packaging.py

```diff
@@ -539,3 +575,258 @@ def plus_one_month(x):
     Utils.check_answer(
         df.select(plus_one_month_udf("a")).collect(), [Row(plus_one_month(d))]
     )
+
+
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC,
+    reason="numpy and pandas are required",
+)
```

Comment:
I don't understand how this reason relates to skipping the test in sprocs. Are numpy and pandas not available in sproc environment?

Reply:
The test for adding packages via requirements also had this, so I maintained the same skip condition and reason - not sure about the purpose.